easybuild/easyconfigs/p/Pysam/Pysam-0.14.1-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# 0.9.1.4:
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+# Modified by: Erich Birngruber
+# Gregor Mendel Institute
+
+easyblock = 'PythonPackage'
+
+name = 'Pysam'
+version = '0.14.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pysam-developers/pysam'
+description = """Pysam is a python module for reading and manipulating Samfiles. 
+ It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/pysam-developers/pysam/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d2bb40cd083c1357768e4683377f03471d160cfe8421136630bfa47f5adb3219']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('ncurses', '6.0'),
+    ('cURL', '7.56.0'),
```

Comment:
@ebirn Please bump to `7.58.0` for compatibility with other easyconfigs using a `*2018*` toolchain